packages/appstore/src/modules/onboarding/onboarding.tsx

```diff
@@ -44,6 +45,9 @@ const Onboarding = observer(({ contents = getTradingHubContents() }: TOnboarding
     const has_active_real_account = useHasActiveRealAccount();
     const steps_list = Object.keys(contents).filter(key => is_mt5_allowed || key !== 'step3');
 
+    // growthbook experiment to skip onboarding for targeted users
+    const growthbook_experiment_skip_onboarding_flow = useFeatureIsOn('skip-onboarding-flow');
```

Comment:
you can use `Analytics.getFeatureState` which returns boolean of feature flag
It's already exist

Reply:
We cannot get realtime updates from growthbook with `Analytics.getFeatureState`